Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -88,9 +121,9 @@ def render_GET(self, request):
         """
         self.events_request = request
 
-        request.write(json.dumps({"type": "events_start"}))
+        request.write(json.dumps({"type": "events_start"}) + '\n')
```

Comment:
I will add it to the description of the events endpoint.

We can also use a protocol where the first 4 bytes indicate the length of the coming JSON dictionary and read it more like a stream but I think this adds unnecessary complexity which we can avoid by simply using a delimiter between the JSON dictionaries.